requirements.txt

```diff
@@ -40,7 +40,7 @@ pymongo==3.7.1
 PyYAML==6.0
 tqdm==4.28.1
 # Python markdown extensions for comment emails
-git+https://github.com/CenterForOpenScience/mdx_del_ins.git
+git+https://github.com/Johnetordoff/mdx_del_ins.git@django-3
```

Comment:
Added to post-release update where we will merge your of `mdx_del_ins.git` and revert the requirement pinging.